### DISCUSSION TITLE
将yolo-world的思想带到分割领域，即Seg-World

### DISCUSSION THREAD

**Discussion by ZJDATY** [AILab-CVC/YOLO-World#273] [Ideas]
当我看到论文中这两张图时，一个大胆的想法在我脑海中孕育，现有的yolo-world 可以实现不同类别框套框的检测，这属于是一个物品，以及其中的子集。现有的分割模型的常规思想，一块区域只能是唯一的类别，而不能从属于另一块大的区域。这若能实现，将是分割模型具有里程碑意义的一步。由此可以引申出Seg-World。这种大胆的想法必须得有一个伟大的团队去实现。
![0ebee61fe4d0cba36c9778be7edd8d6f](https://github.com/AILab-CVC/YOLO-World/assets/29767049/b8ba9581-3b21-4be3-9748-300204e29a4c)
![472ab7718670910201a4c6edc5d5ff91](https://github.com/AILab-CVC/YOLO-World/assets/29767049/7dcf3a8e-c7f3-4144-a501-f1b9c7ef3589)

**Comment by stuqj** [comment]
https://github.com/THU-MIG/yoloe

**Reply by ZJDATY** [reply]
> https://github.com/THU-MIG/yoloe

我的意思是 区域嵌套重合的问题，比如有 人 、人身上的鞋子、裤子 和衣服 ，这四个类别。能否人的分割区域包含所有的物体。
